libs/common/src/sql-db-seeder/utilisation-report/utilisation-report.seed.ts

```diff
@@ -0,0 +1,53 @@
+import { Seeder } from 'typeorm-extension';
+import { DataSource } from 'typeorm';
+import { UtilisationReportEntity } from '../../sql-db-entities';
+import { createNotReceivedReport, createMarkedAsCompletedReport, createUploadedReport } from './utilisation-report.helper';
+import { getCurrentReportPeriodForBankSchedule } from '../../helpers/utilisation-reports';
+import { MOCK_BANKS } from '../../test-helpers/mock-data';
+import { getUsersFromMongoDb } from '../helpers';
+
+export default class UtilisationReportSeeder implements Seeder {
+  /**
+   * Track seeder execution.
+   *
+   * Default: true
+   */
+  track = true;
```

Comment:
Yes - it just adds a table. There is a `config` option for it as well (see `seederOptions`), but you can also define it on individual seeders so that you have a default option and custom otherwise. I'm not strongly attached to defining it on just the config, just the seeder itself or both, so I'm happy to go with whatever pattern you've seen before